base/appointments/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: training
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: training
+  template:
+    metadata:
+      labels:
+        app: training
+    spec:
+      containers:
+      - name: training
+        image: celeo/zdv-easy-appointments:latest
+        imagePullPolicy: Always
+        envFrom:
+        - secretRef:
+          name: training-secrets
+          optional: false
+        ports:
+        - containerPort: 80
+          protocol: TCP
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 80
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 80
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
+          limits:
+            cpu: 50m
+            memory: 256Mi
```

Comment:
Seems low for a PHP site. Have you verified these?

Reply:
![image](https://github.com/kzdv/gitops/assets/772507/53a475a6-51b4-49b0-9ae9-9ef9b1eb6a58)

Initial inspection through `ctop` shows this container fairly lean. I'll poke it a bit more. Raised the limits anyway.